osu.Game.Tests/Editing/Checks/CheckUnsnappedObjectsTest.cs

```diff
@@ -124,32 +125,32 @@ private Mock<Slider> getSliderMock(double startTime, double endTime, int repeats
 
         private void assertOk(List<HitObject> hitobjects)
         {
-            Assert.That(check.Run(getPlayableBeatmap(hitobjects), null), Is.Empty);
+            Assert.That(check.Run(getContext(hitobjects)), Is.Empty);
         }
 
         private void assert1Ms(List<HitObject> hitobjects, int count = 1)
         {
-            var issues = check.Run(getPlayableBeatmap(hitobjects), null).ToList();
+            var issues = check.Run(getContext(hitobjects)).ToList();
 
             Assert.That(issues, Has.Count.EqualTo(count));
             Assert.That(issues.All(issue => issue.Template is CheckUnsnappedObjects.IssueTemplateSmallUnsnap));
         }
 
         private void assert2Ms(List<HitObject> hitobjects, int count = 1)
         {
-            var issues = check.Run(getPlayableBeatmap(hitobjects), null).ToList();
+            var issues = check.Run(getContext(hitobjects)).ToList();
 
             Assert.That(issues, Has.Count.EqualTo(count));
             Assert.That(issues.All(issue => issue.Template is CheckUnsnappedObjects.IssueTemplateLargeUnsnap));
         }
 
-        private IBeatmap getPlayableBeatmap(List<HitObject> hitobjects)
+        private BeatmapVerifierContext getContext(List<HitObject> hitobjects)
         {
-            return new Beatmap<HitObject>
+            return new BeatmapVerifierContext(new Beatmap<HitObject>
             {
                 ControlPointInfo = cpi,
                 HitObjects = hitobjects
-            };
+            }, null);
```

Comment:
This will trigger a warning with the addition of https://github.com/ppy/osu/pull/12801.